### DISCUSSION TITLE
GitHub Sponsors: Get access to Stripe account?

### DISCUSSION THREAD

**Discussion by jgneff** [community/community#23267] [New to GitHub]
<p>I just signed up with GitHub Sponsors, and it seems to have created a <em>Stripe Express</em> account with my e-mail address, but not one that I can actually use to sign in to Stripe. I need access to a full Stripe account for other purposes, but when I try to sign up with my e-mail address, I receive the message “An account already exists with this email.” And when I try to recover the password for the account, I get the message “This account is linked to a business that works with Stripe. To view details, sign in to their website or app directly.” (The GitHub interface doesn’t provide access to the Stripe account password.)</p>
<p>The support person at Stripe suggested that I might be able to change my e-mail address to my personal address through the GitHub interface; sign up with my business e-mail address through Stripe; and then go back to GitHub and switch it back to my business e-mail address. I would try that now, but I’m afraid of making things even worse. In any case, the support representative said I should talk to GitHub.</p>
<p>It seems that my mistake was to sign up for GitHub Sponsors <strong>before</strong> signing up with Stripe directly. Any suggestions on how to recover from that?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jgneff">@jgneff</a>,</p>
<p>Thank you for being here and for signing up for GitHub Sponsors. On our (GitHub’s end) the account that gets created is not connected to any existing accounts. I’ve asked the team, and they think the best solution here would be to use another email address or to use something like <code>existingemailaddress+ghsponsors.</code></p>
<p>You can then input the same bank information if you’d like to have the payout to the same place.</p>

**Comment by jgneff** [comment]
<p>Thank you, <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>. Yes, it seems that using two different e-mail addresses is the only way to enroll in GitHub Sponsors while also having access to a full Stripe account for use in other contexts.</p>
<p>In my case, I just swapped my business e-mail address for my personal one in the <em>Stripe Connect account</em> for GitHub Sponsors. There are multiple places in the GitHub Sponsors dashboard with your e-mail address, but I had to change only the one at the bottom of the GitHub/Stripe page that says “:lock: You are signed in as <em><a href="mailto:name@example.com">name@example.com</a></em>” and has a link to “Edit email or mobile number.”</p>
<p>After switching the Stripe Connect account over to my personal e-mail, I was able to create a full Stripe account with my business e-mail and even connect it to the same checking account.</p>

**Comment by kiwicopple** [comment]
<p>I tried <a class="mention" href="/jgneff">@jgneff</a> 's method and it ended up modifying my email address for the logged in account.</p>
<p><span class="mention">@andreagriffiths11is</span> there a way to reset the form completely and start again? I’m stuck on the “Complete the required documentation to receive payouts” step because the wizard is linked to my logged in account and the org is not appearing in my Stripe dashboard to actually upload the documents.</p>

**Comment by kiwicopple** [comment]
<p>I fixed this after a bit of back-and-forward with Stripe. They did some magic on their side and I could upload the necessary docs</p>
